Author: Steve Beattie <steve.beattie@canonical.com>
Subject: fix -no-pie option to match that actually used by gcc

---
 src/arch/i386/Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: b/src/arch/i386/Makefile
===================================================================
--- a/src/arch/i386/Makefile
+++ b/src/arch/i386/Makefile
@@ -75,8 +75,8 @@ CFLAGS			+= -Ui386
 # output on stderr instead of checking the exit status.
 #
 ifeq ($(CCTYPE),gcc)
-PIE_TEST = [ -z "`$(CC) -fno-PIE -nopie -x c -c /dev/null -o /dev/null 2>&1`" ]
-PIE_FLAGS := $(shell $(PIE_TEST) && $(ECHO) '-fno-PIE -nopie')
+PIE_TEST = [ -z "`$(CC) -fno-PIE -no-pie -x c -c /dev/null -o /dev/null 2>&1`" ]
+PIE_FLAGS := $(shell $(PIE_TEST) && $(ECHO) '-fno-PIE -no-pie')
 WORKAROUND_CFLAGS += $(PIE_FLAGS)
 endif
 
